spring-cloud-stream/src/main/java/org/springframework/cloud/stream/aggregate/AggregateApplication.java

```diff
@@ -73,8 +75,8 @@ static ConfigurableApplicationContext run(Class<?>... apps) {
 	 * @param apps a list of classes, representing root context definitions for apps
 	 * @param args arguments for the apps
 	 */
-	static void runEmbedded(ConfigurableApplicationContext parentContext,
-			Class<?>[] apps, String[][] args) {
+	static void runEmbedded(ConfigurableApplicationContext parentContext, Class<?>[] apps,
```

Comment:
This is an example where keeping things one line would be perfectly acceptable and more readable. I don't think we want to enforce wrapping at 80 characters.